suite-native/module-connect-popup/src/hooks/useConnectParseParams.ts

```diff
@@ -3,10 +3,31 @@ import { useMemo } from 'react';
 import { ParsedURL } from 'expo-linking';
 
 import TrezorConnect from '@trezor/connect';
+import { DEEPLINK_VERSION } from '@trezor/connect/src/data/version';
 
-export const useConnectParseParams = (url: ParsedURL) => {
-    const popupOptions = useMemo(() => {
-        const { queryParams } = url;
+export enum ParseParamsErrors {
+    INVALID_PARAMS = 'invalid_params',
+    VERSION_UNSUPPORTED = 'version_unsupported',
```

Comment:
I don't see any error related to version in connect errors, but maybe we could add it there https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/constants/errors.ts#L0-L1 ?